File: evidence/generated-systems/angular/aerospace-on-angular/src/app/models/AircraftPackage.js

// Define collection and schema for AircraftPackage
export interface AircraftPackage {
    name:
	type : string
    Options:
 	type : [{ type: Schema.Types.ObjectId, ref: 'AircraftOption' }]
    Variants:
 	type : [{ type: Schema.Types.ObjectId, ref: 'AircraftVariant' }]
    PackageType:
 	type : String
#
    collection: 'aircraftPackages'
}

File: evidence/generated-systems/rubyonrails/aerospace-on-rails/aerospaceonrails/app/models/aircraftpackage.rb
class AircraftPackage < ApplicationRecord
  enum PackageType: [:PerformancePack, :CabinPack, :ConnectivityPack, :CompliancePack]


  has_many :Options, class_name: 'AircraftOption'
  has_many :Variants, class_name: 'AircraftVariant'

end
